pkg/base1/cockpit.js

```diff
@@ -24,6 +24,7 @@
 import "../lib/patternfly/patternfly-cockpit.scss";
 import "../lib/page.scss";
 import "@patternfly/patternfly/components/Button/button.css";
+import "../lib/table.css";
```

Comment:
Now I see what triggered your question from this morning. As I said, I'd be happy to just leave the examples "ugly" (they already are, anyway), and not support cockpit.css any further. But fine for now.